extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/mpmetrics/CountedInterceptor.java

```diff
@@ -24,22 +24,17 @@
     }
 
     @AroundConstruct
-    Object countedConstructor(InvocationContext context) throws Exception {
+    Object countedConstructor(ArcInvocationContext context) throws Exception {
         return increment(context, context.getConstructor().getDeclaringClass().getSimpleName());
     }
 
     @AroundInvoke
-    Object countedMethod(InvocationContext context) throws Exception {
-        return increment(context, context.getMethod().getName());
-    }
-
```

Comment:
was it intentional to drop this method?

Reply:
Yes, `@AroundTimeout` methods are bound to the EJB timer service and thus are not supported in CDI (and quarkus).